### PR TITLE
Lk scalable optimus doc

### PR DESCRIPTION
**Pull request by ekiernan**
- Updated Optimus Overview for scalable Optimus (this PR: https://github.com/broadinstitute/warp/pull/593/files).
- Updated Methods, Matrix overview, and diagram

### CODE REVIEW

File: website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 1]
```diff
@@ -9,15 +9,18 @@ The Optimus pipeline's default count matrix output is a Loom file, an HDF5 file
 
 The matrix contains global attributes detailing how counts were generated for the single-cell or single-nucleus parameters ([Table 1](#table-1-global-attributes)). It additionally contains UMI-corrected counts as well as multiple metrics for both individual cells (the columns of the matrix; [Table 2](#table-2-column-attributes-cell-metrics)) and individual genes (the rows of the matrix; [Table 3](#table-3-row-attributes-gene-metrics)). 
 
-The tables below document these metrics, list which tools generate them, and define them.
+The type of gene counts in the Loom will vary depending on Optimus workflow counting_mode. If running single-cell data (sc_rna mode), the counts will include only exonic gene counts. If running single-nucleus data (sn_rna mode), the counts will be whole transcript.
```
Comment (ekiernan) [inline website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md:12]:
```suggestion
The type of gene counts in the Loom will vary depending on the Optimus workflow counting_mode. If running single-cell data (sc_rna mode), the counts will include only exonic gene counts. If running single-nucleus data (sn_rna mode), the counts will be whole transcript.

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 2]
```diff
@@ -199,14 +205,16 @@ EmptyDrops relies on a visual knee point inflection (described in [Lun et al. (2
 Often snRNAseq data does not produce a visual knee point inflection when running emptyDrops and the tool can not accurately distinguish ambient-like cells from empty droplets. For this reason, emptyDrops is not used if Optimus counting_mode is set to `sn_rna`, and the output Loom matrix will not contain emptyDrops metrics.
 :::
 
-
-#### 5.  Matrix construction
+#### 7.  Matrix construction
 
 The [OptimusLoomGeneration](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/LoomUtils.wdl) task uses a custom python script to merge the converted STARsolo count matrix, the emptyDrops results, and the cell and gene metrics into a Loom-formatted cell-by-gene matrix. Read full details for all the metrics in the [Optimus Count Matrix Overview](./Loom_schema.md).
 
+The type of gene counts in the Loom will vary depending on Optimus workflow counting_mode. If running single-cell data (sc_rna mode), the counts will include only exonic gene counts. If running single-nucleus data (sn_rna mode), the counts will be whole transcript.
```
Comment (ekiernan) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:212]:
```suggestion
The type of gene counts in the Loom will vary depending on the Optimus workflow counting_mode. If running single-cell data (sc_rna mode), the counts will include only exonic gene counts. If running single-nucleus data (sn_rna mode), the counts will be whole transcript.

**Review by kayleemathews — APPROVED** [review]
A few minor things but looks good! Thanks Liz!

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 3]
```diff
@@ -29,10 +29,10 @@ The following table provides a quick glance at the Optimus pipeline features:
...
 | Workflow language | WDL 1.0 | [openWDL](https://github.com/openwdl/wdl) |
-| Genomic Reference Sequence| GRCh38 human genome primary sequence and M21 (GRCm38.p6) mouse genome primary sequence | GENCODE [Human](https://www.gencodegenes.org/human/release_27.html) and [Mouse](https://www.gencodegenes.org/mouse/release_M21.html)
+| Genomic Reference Sequence | GRCh38 human genome primary sequence and M21 (GRCm38.p6) mouse genome primary sequence | GENCODE [Human](https://www.gencodegenes.org/human/release_27.html) and [Mouse](https://www.gencodegenes.org/mouse/release_M21.html)
 | Transcriptomic reference annotation | V27 GENCODE human transcriptome and M21 mouse transcriptome | GENCODE [Human](ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.annotation.gtf.gz) and [Mouse](ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M21/gencode.vM21.annotation.gff3.gz) |
 | Aligner and transcript quantification | STARsolo (v.2.7.9a) | [Dobin, et al.,2021](https://www.biorxiv.org/content/10.1101/2021.05.05.442755v1) |
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/Pipelines/Optimus_Pipeline/README.md]: Done.

**Pull request merged**